packages/mobile/src/navigator/Navigator.tsx

```diff
@@ -112,6 +112,7 @@ export const modalScreenOptions = ({ route, navigation }: NavigationOptions) =>
       gestureEnabled: true,
       cardOverlayEnabled: true,
       headerStatusBarHeight:
+        // @ts-ignore
```

Comment:
FIXME

Reply:
leaving this with the ts-ignore for now because I think the library fixed it, but we need to test.

Reply:
What's the error?

Reply:
`var route: Route<string, unknown>
Argument of type 'Route<string, unknown>' is not assignable to parameter of type 'NavigationRoute<StackParamList, Screens.AccountKeyEducation | Screens.BackupComplete | Screens.BackupIntroduction | Screens.BackupPhrase | Screens.BackupQuiz | Screens.DappKitAccountAuth | Screens.DappKitSignTxScreen | Screens.DappKitTxDataScreen | ... 57 more ... | Screens.WithdrawCeloScreen>'.
  Type 'Route<string, unknown>' is not assignable to type 'Readonly<{ key: string; name: Screens.AccountKeyEducation | Screens.BackupComplete | Screens.BackupIntroduction | Screens.BackupPhrase | Screens.BackupQuiz | Screens.DappKitAccountAuth | Screens.DappKitSignTxScreen | Screens.DappKitTxDataScreen | ... 57 more ... | Screens.WithdrawCeloScreen; }>'.
    Types of property 'name' are incompatible.
      Type 'string' is not assignable to type 'Screens.AccountKeyEducation | Screens.BackupComplete | Screens.BackupIntroduction | Screens.BackupPhrase | Screens.BackupQuiz | Screens.DappKitAccountAuth | Screens.DappKitSignTxScreen | Screens.DappKitTxDataScreen | Screens.Debug | ... 56 more ... | Screens.WithdrawCeloScreen'.ts(2345)
`

Reply:
Thanks, I can have a look tomorrow morning.

Reply:
Fixed 🙂

Reply:
awesome! thank you!